src/TraceMiddleware.php

```diff
@@ -40,6 +42,13 @@ class TraceMiddleware
         '/\["key.{27,36}Stream.{9}\]=>\s+.{7}\d{2}\) "\X{16,64}"/U' => '["key":[CONTENT KEY]]',
     ];
 
+    private static $sensitiveContext = [
+        'resolved_identity' => [
+            AwsCredentialIdentity::class => true,
+            BearerTokenIdentity::class => true
```

Comment:
Should we add the interface here that these subclasses implements instead?, just because it is more explicit in which type of objects should be considered as sensitive.

Reply:
What would that look like? I think it's fine to show which identity has been selected- I'm only concerned about exposing auth strings